llvm/test/SYCLPropagateAspectUsage/output.ll.ll

```diff
@@ -0,0 +1,22 @@
+; ModuleID = 'output.ll'
```

Comment:
This file should be removed, I suppose. As well as `llvm/test/SYCLPropagateAspectUsage/output.ll`